mon-pix/tests/unit/components/campaigns/assessment/results/evaluation-results-tabs/index-test.js

```diff
@@ -0,0 +1,88 @@
+import { setupTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+import createGlimmerComponent from '../../../../../../helpers/create-glimmer-component';
+
+module(
+  'Unit | Component | Campaigns | Assessment | Results | EvaluationResultsTabs | ResultsDetails | CompetenceRow',
+  function (hooks) {
+    setupTest(hooks);
+
+    module('#showRewardsTab', function () {
+      module('when there is no badge', function () {
+        test('should return false', async function (assert) {
+          // given
+          const component = createGlimmerComponent('campaigns/assessment/results/evaluation-results-tabs/index');
+
+          component.args.campaignParticipationResult = {
+            campaignParticipationBadges: [],
+          };
+
+          // then
+          assert.false(component.showRewardsTab);
+        });
+      });
+
+      module('when there is badges', function () {
+        module('when there is only not-acquired and not always visible badge', function () {
```

Comment:
module('when there is only not-acquired and no always visible badge', function () {